src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -301,9 +302,11 @@ public Map<String, String> getRecipientsOfQuestion(
         switch (recipientType) {
         case SELF:
             if (question.getGiverType() == FeedbackParticipantType.TEAMS) {
-                recipients.put(giverTeam, giverTeam);
+                recipients.put(giverTeam,
+                       new FeedbackQuestionRecipient(giverTeam, giverTeam, null, null));
```

Comment:
This is fine, but having another constructor (like what you did before) can help to save from the repetitive 2x`null`s. In fact I didn't expect that to be removed?

Reply:
I think I have misunderstood. I will make the changes again